Makefile

```diff
@@ -18,8 +18,9 @@ BUILD_PLATFORMS ?= linux/amd64
 all: manager
 
 # Run tests
-test: generate fmt vet manifests api-docs
-	go test ./... -coverprofile cover.out
+KUBEBUILDER_ASSETS?="$(shell $(ENVTEST) use -i $(ENVTEST_KUBERNETES_VERSION) --bin-dir=$(ENVTEST_ASSETS_DIR) -p path)"
```

Comment:
```suggestion
KUBEBUILDER_ASSETS?="$(shell $(ENVTEST) --arch=amd64 use -i $(ENVTEST_KUBERNETES_VERSION) --bin-dir=$(ENVTEST_ASSETS_DIR) -p path)"
```

Reply:
@stefanprodan I will of course do it... but what about 
```
--arch=${GOARCH}
```

would it solve the problem, or do we need to hard code amd64?

Reply:
The problem is that arm64 envtest images are at present not available.

Reply:
got it, thank you for the clarification